samples/star-wars-api/HttpHandlers.fs

```diff
@@ -55,52 +57,110 @@ module HttpHandlers =
             let toResponse { DocumentId = documentId; Content = content; Metadata = metadata } =
                 match content with
                 | Direct (data, errs) ->
-                    logger.LogInformation($"Produced direct GraphQL response with documentId = '{{documentId}}' and metadata:{Environment.NewLine}{{metadata}}", documentId, metadata)
+                    logger.LogInformation (
+                        $"Produced direct GraphQL response with documentId = '{{documentId}}' and metadata:{Environment.NewLine}{{metadata}}",
+                        documentId,
+                        metadata
+                    )
+
                     if logger.IsEnabled LogLevel.Trace then
-                        logger.LogTrace($"GraphQL response data:{Environment.NewLine}:{{data}}", JsonSerializer.Serialize(data, jsonSerializerOptions))
+                        logger.LogTrace (
+                            $"GraphQL response data:{Environment.NewLine}{{data}}",
+                            serializeIt data
+                        )
+
                     GQLResponse.Direct (documentId, data, errs)
 
                 | Deferred (data, errs, deferred) ->
-                    logger.LogInformation($"Produced deferred GraphQL response with documentId = '{{documentId}}' and metadata:{Environment.NewLine}{{metadata}}", documentId, metadata)
+                    logger.LogInformation (
+                        $"Produced deferred GraphQL response with documentId = '{{documentId}}' and metadata:{Environment.NewLine}{{metadata}}",
+                        documentId,
+                        metadata
+                    )
+                    if logger.IsEnabled LogLevel.Trace && errs.Length > 0 then
+                        logger.LogTrace ($"{{number}} errors:{Environment.NewLine}{{errs}}", errs.Length, errs)
```

Comment:
What about the same for direct?
We can move this code into a function as it duplicates

Reply:
"if logger.IsEnabled LogLevel.Trace" seems redundant to me.  Doesn't calling "logger.LogTrace" when "LogLevel.Trace" is false, just skip logging that line anyway?  How about if I just remove "if logger.IsEnabled LogLevel.Trace" everywhere?